### PR TITLE
made it bad

### PR DESCRIPTION
**Pull request by DefectivenessYT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,12 @@
...
-I'm Jacques, but I'm also known as Hex or Jack. I'm a 15 year old "kinda good" Node.js developer who loves to code in their spare time.
+I'm Ben, but I'm also known as Defectiveness or benne. I'm 15 and do stupid terrible code in my spare time.
 
 ## A couple projects I'm proud of
 <!--Name - Description - My role<br>-->
-[Jinkaku](https://top.gg/bot/706271163168587848) - A discord bot for roleplayers - Owner and Lead Dev<br>
-[KeyFM](https://keyfm.net) - An online radio station - Administrator<br>
-[CloudX](https://cloudx.wtf) - An online radio station - Bot Developer<br>
-Music4All - A discord bot which played music. My first ever Discord bot. Shutdown in 2018 - Owner and Lead Dev
+[Dual](https://dual.pw) - Online Radio Station - Owner<br>
```
Comment (HexDevv) [inline README.md:7]: i dont work here

File: README.md [hunk 2]
```diff
@@ -1,29 +1,12 @@
...
+[Securio](https://securio.tech) - A DDoS Protection and web performance company - Support Staff<br>
+[Xonos](https://xonos.host) - Game and Web hosting company - Support Staff and Developer<br>
```
Comment (HexDevv) [inline README.md:9]: neither do i work here

File: README.md [hunk 3]
```diff
@@ -1,29 +1,12 @@
 <h1 align="center">Hi there 👋</h1>
 
-I'm Jacques, but I'm also known as Hex or Jack. I'm a 15 year old "kinda good" Node.js developer who loves to code in their spare time.
+I'm Ben, but I'm also known as Defectiveness or benne. I'm 15 and do stupid terrible code in my spare time.
```
Comment (HexDevv) [inline README.md:3]: my name isn't ben

File: README.md [hunk 4]
```diff
@@ -1,29 +1,12 @@
...
 
 ## A couple projects I'm proud of
 <!--Name - Description - My role<br>-->
-[Jinkaku](https://top.gg/bot/706271163168587848) - A discord bot for roleplayers - Owner and Lead Dev<br>
-[KeyFM](https://keyfm.net) - An online radio station - Administrator<br>
-[CloudX](https://cloudx.wtf) - An online radio station - Bot Developer<br>
-Music4All - A discord bot which played music. My first ever Discord bot. Shutdown in 2018 - Owner and Lead Dev
+[Dual](https://dual.pw) - Online Radio Station - Owner<br>
+[Securio](https://securio.tech) - A DDoS Protection and web performance company - Support Staff<br>
```
Comment (HexDevv) [inline README.md:8]: i think this is closed

File: README.md [hunk 5]
```diff
@@ -1,29 +1,12 @@
...
-- 🔭 I’m currently working on ...
-- 🌱 I’m currently learning ...
-- 👯 I’m looking to collaborate on ...
-- 🤔 I’m looking for help with ...
-- 💬 Ask me about ...
-- 📫 How to reach me: ...
-- 😄 Pronouns: ...
-- ⚡ Fun fact: ...
--->
+[![Github Stats](https://github-readme-stats.vercel.app/api?username=defectivenessyt)]
```
Comment (HexDevv) [inline README.md:12]: this isn't my username

**Comment by DefectivenessYT** [comment]
yeah but like fuck off

File: README.md (hunk 1 above)
Comment (DefectivenessYT) [reply README.md:7]: i mean you **could** change that

Comment (DamonOnYT) [reply README.md:7]: bad

Comment (DamonOnYT) [reply README.md:7]: how do i block someone on github

Comment (DefectivenessYT) [reply README.md:7]: uh you go to https://dual.pw and tune in

Comment (HexDevv) [reply README.md:7]: idk

File: README.md [hunk 6]
```diff
@@ -1,29 +1,15 @@
...
-- 🔭 I’m currently working on ...
-- 🌱 I’m currently learning ...
-- 👯 I’m looking to collaborate on ...
-- 🤔 I’m looking for help with ...
-- 💬 Ask me about ...
-- 📫 How to reach me: ...
-- 😄 Pronouns: ...
-- ⚡ Fun fact: ...
--->
+grwepoijfrejgiperjigphojipotehjipoterhjpio[etrhsajipoetrhajipobngk
```
Comment (HexDevv) [inline README.md:15]: yes

**Review by HexDevv — CHANGES_REQUESTED** [review]

**Comment by HexDevv** [comment]
Closed. Merged with #2

**Pull request closed without merging**